deploy/helm/sumologic/values.yaml

```diff
@@ -79,6 +79,13 @@ sumologic:
         helm.sh/hook-weight: "2"
         helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
     job:
+      resources:
```

Comment:
Have we tested that these are reasonable limits? The setup job really doesn't do a lot, so these seem like they might be high?

Reply:
I've tried to look at setup job's utilization and I've noticed values of used memory around 90-120Mi.

Reply:
Thanks @pmalek-sumo for verifying that. I think then the request limit of `128 Mi` is apt.